pnpm-lock.yaml

```diff
@@ -0,0 +1,16050 @@
+lockfileVersion: 5.4
+
+specifiers:
+  '@commitlint/cli': ^17.6.7
```

Comment:
What's the reason to move to `pnpm`?

Reply:
the dependency tree was a mess, it helps a lot moving to pnpm, and since internally we were discussing to give a try, we decided, why not giving a try